app/core/ModuleLoader.coffee

```diff
@@ -63,6 +63,7 @@ module.exports = ModuleLoader = class ModuleLoader extends CocoClass
 
     else if path is "esper"
       try
+        # TODO Webpack: Do this in the new setup?
```

Comment:
Not sure if this section should be a blocker, or what exactly it does.

Reply:
Let's try to port this code.

Reply:
Using the compatible version of esper is a pretty big performance hit.